tests/suites/test_suite_bignum_core.function

```diff
@@ -941,3 +941,51 @@ exit:
     mbedtls_mpi_free( &RR_REF );
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void mpi_core_ct_uint_table_lookup( int bitlen, int window_size )
+{
+    size_t limbs = BITS_TO_LIMBS( bitlen );
+    size_t count = ( (size_t) 1 ) << window_size;
```

Comment:
I don't understand why this is a window size. Just seems to be log2 of number of entries we want in the table. What am I missing?

Reply:
In the exponentiation algorithm we use the lookup table size depends on the number of bits of the exponent for which we do the pre-calculations and the lookup.